common/components/whats-new-banner/template.njk

```diff
@@ -1,14 +1,16 @@
-<div id="whats-new-banner" class="whats-new-banner" data-module="app-whats-new-banner" data-content-date="{{ params.date }}">
-  <div class="govuk-width-container">
-    <div class="govuk-grid-row banner-padding">
-      <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">{{ params.title }}</h1>
-        <p class="govuk-body">{{ params.date }}: {{ params.body }}.<p>
-        <a href="/whats-new" id="whats-new-page-link" class="govuk-link">Read more about these changes.</a>
-      </div>
-      <div class="govuk-grid-column-one-third">
-        <a href="#" id="dismiss-banner-button" class="govuk-link dismiss-button">Dismiss</a>
+{% if not params.error %}
```

Comment:
I wonder if we could do `if params.title && params.date && params.body` here, which would support the error case but also allow us to better handle the case when there are no entries, and therefore no banner to show?